modules/assignment/src/main/java/com/privacy/wasabi/assignment/MongoAssignmentIngestionExecutor.java

```diff
@@ -0,0 +1,84 @@
+package com.privacy.wasabi.assignment;
+
+import com.intuit.wasabi.assignment.AssignmentIngestionExecutor;
+import com.intuit.wasabi.assignmentobjects.AssignmentEnvelopePayload;
+import com.intuit.wasabi.assignmentobjects.User;
+import com.intuit.wasabi.assignmentobjects.User.ID;
+import com.intuit.wasabi.experimentobjects.Bucket;
+import com.intuit.wasabi.experimentobjects.Experiment;
+import com.intuit.wasabi.experimentobjects.Experiment.Label;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.DB;
+import com.mongodb.DBCollection;
+import com.mongodb.BasicDBObject;
+
+import org.slf4j.Logger;
+import static org.slf4j.LoggerFactory.getLogger;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.concurrent.Future;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.Date;
+
+import java.net.UnknownHostException;
+
+public class MongoAssignmentIngestionExecutor implements AssignmentIngestionExecutor {
+
+    public static final String NAME = "MONGOINGESTOR";
+    private static final Logger LOGGER = getLogger(MongoAssignmentIngestionExecutor.class);
+
+    private ExecutorService executor = Executors.newSingleThreadExecutor();
+
+    @Override
+    public Future<?> execute(AssignmentEnvelopePayload assignmentEnvelopePayload) {
+        return executor.submit(() -> {
+            String mongoURI = System.getenv("MONGO_URI");
+            String mongoDB = System.getenv("MONGO_DB");
+            if (mongoURI != null && mongoDB != null) {
+                try {
+                    MongoClient mongoClient = new MongoClient(new MongoClientURI(mongoURI));
+                    DB database = mongoClient.getDB(mongoDB);
+                    DBCollection assignmentCollection = database.getCollection("experimentassignments");
```

Comment:
I was initially thinking we'd insert this into the user document, but perhaps this is good. We can always have `mosql` (our mongo -> pg replication mechanism) replicate over a different collection.

Reply:
@mnisjk you have any preference on this?

Reply:
Got it.